src/classes/Collision/GjkEpa.js

```diff
@@ -737,3 +778,20 @@ Goblin.GjkEpa.Face.prototype = {
         }
     };
 } )();
+
+/**
+ * Wraps a vector in a proxy object so that it can ne used in GJK / EPA
+ * @param {Goblin.Vector3} point
+ * @constructor
+ */
+Goblin.GjkEpa.PointProxy = function( point ) {
+    this.position = point;
```

Comment:
Yes, it's safe because we're using it only inside GJK (if it modified object's position, we would be in a big trouble). Considering this, there is no issues with hidden class either